legal-api/src/legal_api/reports/business_document.py

```diff
@@ -411,6 +411,10 @@ def _set_liquidation_details(self, business: dict):
 
     @staticmethod
     def _format_address(address):
+        address['streetAddressAdditional'] = address.get('streetAddressAdditional') or ''
+        address['addressRegion'] = address.get('addressRegion') or ''
```

Comment:
`addressRegion` is a [required property](https://github.com/bcgov/business-schemas/blob/35a99eabfa8b70527124f90c4b6807d159aea3b4/src/registry_schemas/schemas/address.json#L47) so you shouldn't need to do this. If you do then there's a data error(s).

Reply:
as per the schema it can be null https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/address.json#L23

Reply:
Oh, so it has to be present but can be null? That sounds like "not required" to me, haha.

Any way we can update the schema to be consistent?

Reply:
One thing we could do is remove `addressRegion` from required and allow null in `streetAddressAdditional`

Reply:
That makes sense to me.